api/v1alpha1/eip_types.go

```diff
@@ -25,6 +25,7 @@ type EipSpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^([0-9]{1,3}\.){3}[0-9]{1,3}((\/([0-9]|[1-2][0-9]|3[0-2]))|(\-([0-9]{1,3}\.){3}[0-9]{1,3}))?$`
 	Address       string `json:"address,omitempty"`
+	Protocol      string `json:"protocol,omitempty"`
```

Comment:
Should we update the version to v1alpha2?

Reply:
the version update could be done later,  I have no idea on how it works now.